spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/CassandraAdminOperations.java

```diff
@@ -46,15 +47,31 @@ public interface CassandraAdminOperations extends CassandraOperations {
 	 * parameter {@code ifNotExists} is {@literal true}, this is a no-op and {@literal false} is returned. If the table
 	 * doesn't exist, parameter {@code ifNotExists} is ignored, the table is created and {@literal true} is returned.
 	 *
+	 * @deprecated Deprecated in favor of {@link #createTable(boolean, Class, Map)}, since we can infer the table name from
+	 *             passed entity class
 	 * @param ifNotExists If true, will only create the table if it doesn't exist, else the create operation will be
 	 *          ignored.
 	 * @param tableName The name of the table.
 	 * @param entityClass The class whose fields determine the columns created.
 	 * @param optionsByName Table options, given by the string option name and the appropriate option value.
 	 */
+	@Deprecated(forRemoval = true)
```

Comment:
We do not intent deprecating this method as a table can be created from the given `Class` using a different table name.